### PR TITLE
Discovery Pane Content Update November 2016.

### LINKED ISSUE #3931: Disco Pane content update - Nov. 2016
For the Thursday, **17 November** push, can we please REMOVE: 

- Emoji Cheatsheet
- All three themes (glitter closeup, metal flowers, Live With Music)

… and replace them with OmniSidebar and three new themes, respectively (note full copy included):

**Easily access bookmarks**
with [OmniSidebar by Quicksaver](https://addons.mozilla.org/firefox/addon/omnisidebar/)
Are you constantly scrolling through your bookmarks? Bring your lists into view with a single, simple gesture. 

[themes]

Aurora Australis by [silfiriel](https://addons.mozilla.org/firefox/addon/aurora-australis/)

snow style by [elakeduck](https://addons.mozilla.org/firefox/addon/snow-style/)

My Vinyl by [baliclem](https://addons.mozilla.org/firefox/addon/my-vinyl/)

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Fixes #3931

For QA (and review): I am working on a script that populates the appropriate data on dev and stage for testing but that needs a few more minutes / hours of fiddling around so in the mean-time the best thing to do is to verify the add-on ids in this pull request point to the correct add-ons and themes when called with `https://addons.mozilla.org/en-US/firefox/addon/<addon-id>/`

### CODE REVIEW

**Comment by EnTeQuAk** [comment]
Once merged I'll run another l10n extraction.

File: src/olympia/discovery/data.py [hunk 1]
```diff
@@ -30,33 +31,33 @@ def __init__(self, *args, **kwargs):
...
         description=string_concat(
             '<blockquote>',
             _(u'So much more than just a screenshot tool, this add-on also '
               u'lets you edit, annotate, and share images.'),
             '</blockquote>')),
 
-    # Metal Flowers - animated
-    DiscoItem(addon_id=188332),
+    # Theme: Snow Style
+    DiscoItem(addon_id=68349, addon_name=_(u'Snow Style')),
```
Comment (diox) [inline src/olympia/discovery/data.py]: I don't think we want to translate that, since the original add-on name isn't translated.

Comment (EnTeQuAk) [inline src/olympia/discovery/data.py]: Fixed in  	1e12501

**Comment by muffinresearch** [comment]
@EnTeQuAk the plan is to get this landed, do the extraction and Scott is looking at getting translations from another source today. I should know tomorrow if we have the translations and can land them manually. In the meantime please land this and run the extraction asap. 

We'll work out the process for what gets cherry picked and how to deal with making the manual localization updates tomorrow AM (wednesday).

/cc @diox

**Pull request merged**